.github/workflows/blade-chromatic.yml

```diff
@@ -32,3 +32,4 @@ jobs:
           exitOnceUploaded: true
         env:
           GITHUB_SHA: ${{ github.sha }}
```

Comment:
can we just do a conditional in actions itself? where if the ref is master then we set the GitHub Sha to empty or null instead of passing everything and then checking in the code? with this we won't even need to add more conditions to our code and it'll also fit your logic intent that if you sha we'll use that else just use `*`

Reply:
Exactly what I tried first but tricky part is we can't do ternary operators in github actions so we'll have to do something like - 

```yaml
- name: Conditionally set env variable
  if: github.ref !== '/refs/heads/master'
  run: |
    echo "GITHUB_SHA=${{ github.sha }}" >> $GITHUB_ENV
```

seems hackier than just passing the ref forward and checking in code 🙈